### DISCUSSION TITLE
Central switch for turning on/off

### DISCUSSION THREAD

**Discussion by boroczkigabor** [jmcollin78/versatile_thermostat#517] [General]
Hi,

I would like to have a central switch (that would most likely be a binary_sensor at the end) which enables/disables VTherms. I.e. only turn on heating when a given condition is true.
Use case: I have a heat pump which is controlled by a central, non-smart thermostat. I have a few VTherms controlling valves in rooms, however it may happen that the room would require heating but the living room where the thermostat is, don't. In this particular case, VTherm turns on even though there will be no actual heating happening.
Another use case I can think of, to turn on electric heatings in countries where power pricing is dynamically changing, and therefore only turn on a certain VTherm when price is low. 

Any thoughts?

**Comment by jmcollin78** [comment]
Hello @boroczkigabor ,

It seems you are looking for the central control of VTherms. See readme for more informations.
